docs/zkapps/advanced-snarkyjs/permissions.mdx

```diff
@@ -178,12 +167,11 @@ tx = await Mina.transaction(account1Address, () => {
 await tx.sign([account1Key]).send();
 ```
 
-This example transaction creates a new account update for our smart contract address. Right after that, the new account update sends 1 MINA to the address of the fee payer (`account1Address`).
+This malicious example transaction creates a new account update for our smart contract address. Right after that, the new account update sends 1 MINA to the address of the fee payer (`account1Address`).
 At the end of the transaction block, the transaction is signed only with the private key of the fee payer - not the private key of the smart contract!
 Because the permissions for sending funds away from a smart contract are set to `none`, this transaction succeeds and drains 1 MINA from the smart contract.
 
-Now, if you change the `send` permission to something else, like `signature`, the manual account update fails with `Update_not_permitted_balance` which does not allow to withdraw funds from the smart contract,
-since the authorization does not fit the permission for `send` that now requires a valid signature!
+Now, if you change the `send` permission to something else, like `signature`, the manual account update fails with `Update_not_permitted_balance` that prevents withdrawing funds from the smart contract since the authorization does not fit the permission for `send` that now requires a valid signature!
```

Comment:
```suggestion
Now, if you change the `send` permission to something else, like `signature`, the manual account update fails with `Update_not_permitted_balance` that prevents withdrawing funds from the smart contract since the authorization does not fit the permission for `send` that now requires a valid signature.
```